bigbluebutton-client/src/org/bigbluebutton/main/model/users/Conference.as

```diff
@@ -197,8 +198,16 @@ package org.bigbluebutton.main.model.users {
 				}
 			}
 			return null;
-		}
-		
+        }
+
+        public function userIsModerator(userId:String):Boolean {
+            var moderator:BBBUser = getTheOnlyModerator();

```

Comment:
You get the first moderator via getTheOnlyModerator(). However, if you have 2+ moderators here you will have userIsModerator(userId) return false while the user's role is moderator.
Why not var moderator:BBBUser = getUser(userId); return moderator.role == Role.MODERATOR; return false;}?